exporting/prometheus/multi_metadata.yaml

```diff
@@ -0,0 +1,436 @@
+            required: true
+            detailed_description: |
+              The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+              - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+              - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+              - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+  
+              Example IPv4:
+                ```yaml
+                destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
+                ```
+              Example IPv6 and IPv4 together:
+              ```yaml
+              destination = [ffff:...:0001]:2003 10.11.12.1:2003
+              ```
+              When multiple servers are defined, Netdata will try the next one when the previous one fails.
+          - name: 'username'
+            default_value: 'my_username'
+            description: 'Username for HTTP authentication'
+            required: false
+          - name: 'password'
+            default_value: 'my_password'
+            description: 'Password for HTTP authentication'
+            required: false
+          - name: 'data source'
+            default_value: ''
+            description: 'Selects the kind of data that will be sent to the external database. (as collected|average|sum)'
+            required: false
+          - name: 'hostname'
+            default_value: '[global].hostname'
+            description: 'The hostname to be used for sending data to the external database server.'
+            required: false
+          - name: 'prefix'
+            default_value: 'Netdata'
+            description: 'The prefix to add to all metrics.'
+            required: false
+          - name: 'update every'
+            default_value: '10'
+            description: |
+              Frequency of sending sending data to the external database, in seconds.
+            required: false
+            detailed_description: |
+              Netdata will add some randomness to this number, to prevent stressing the external server when many Netdata servers
+              send data to the same database. This randomness does not affect the quality of the data, only the time they are sent.
+          - name: 'buffer on failures'
+            default_value: '10'
+            description: |
+              The number of iterations (`update every` seconds) to buffer data, when the external database server is not available. 
+            required: false
+            detailed_description: |
+              If the server fails to receive the data after that many failures, data loss on the connector instance is expected (Netdata will also log it).
+          - name: 'timeout ms'
+            default_value: '2 * update_every * 1000'
```

Comment:
I think to show value like this can create confusion, instead I suggest to use the default value : 

```sh
timeout ms = 20000
```
, and explain details in description.